test/language.js

```diff
@@ -86,7 +86,23 @@ describe('language()', () => {
     it('ignores preference case when matching', () => {
 
         const language = Accept.language('da, en-GB, en', ['en-us', 'en-gb']); // en-GB vs en-gb
-        expect(language).to.equal('en-gb');
+        expect(language).to.equal('en-us');
+    });
+
+    it('returns language using range match', () => {
+
+        expect(Accept.language('da', ['da-DK'])).to.equal('da-DK');
+        expect(Accept.language('en-US, en', ['en-GB', 'en-US'])).to.equal('en-GB');
```

Comment:
While I see how this result satisfies the spec, this one looks a bit odd at first.  Do we feel good about the order of the language tags being a better arbitrary tie-breaker than specificity?

Reply:
Also, is there a reason "explicit preference overrides range match" shouldn't come into play here?

Reply:
Yes, explicitness doesn't really mean anything. Since the client gave them the same weight, it doesn't care which the server choses.